tests/tensor/test_blas_scipy.py

```diff
@@ -58,6 +60,14 @@ def test_scaled_A_plus_scaled_outer(self):
         self.assertFunctionContains(f, gemm_no_inplace)
         self.run_f(f)  # DebugMode tests correctness
 
+    def test_pickle(self):
+        out = ScipyGer(destructive=False)(self.A, self.a, self.x, self.y)
+        f = pytensor.function([self.A, self.a, self.x, self.y], out)
+
+        assert isinstance(
```

Comment:
Since we are already compiling the function might also evaluate it, just to double check it works after pickling?

Reply:
Ok, I’ll add that in tomorrow morning